frontend/src/components/mine/MineDashboard.js

```diff
@@ -148,7 +150,7 @@ export class MineDashboard extends Component {
                 </TabPane>
                 <TabPane tab="Permit" key="permit">
                   <div className="tab__content">
-                    <MinePermitInfo mine={mine} />
+                    <MinePermitInfo mine={mine} {...this.props} />
```

Comment:
I know we do this all over the codebase, so it's not a blocking change, but I'd like to propose that we start passing props in explicitly as a pattern going forward. It's a team decision that requires a bit of discussion, but the existing pattern of passing in all props like this is generally considered an anti-pattern in React.

Reply:
I'm all for it and I don't like passing in this.props, but I'll address this in a future PR we are immediately following this up with more changes to these components.

Reply:
Definitely not in scope for this PR. Happy to hear we're on the same page with it. Marking as resolved and we can revisit this as we touch components going forward